app/assets/stylesheets/common.scss

```diff
@@ -183,3 +183,25 @@ input.has-error {
   }
 }
 
+#new_page .form-inline input[type='text'] {
+  width: 400px;
+}
+
+.loading {
+  margin: 0 5px;
+  .glyphicon-refresh-animate {
+      -animation: spin .7s infinite linear;
+      -webkit-animation: spin2 .7s infinite linear;
+  }
+
+  @-webkit-keyframes spin2 {
+      from { -webkit-transform: rotate(0deg);}
+      to { -webkit-transform: rotate(360deg);}
+  }
+
+  @keyframes spin {
+      from { transform: scale(1) rotate(0deg);}
+      to { transform: scale(1) rotate(360deg);}
+  }
+}
```

Comment:
you're missing the -moz- and -o- vendor prefixes. here I would recommend using compass's animation mixin to just get all the vendor prefixes for free.